### ISSUE TITLE
Error page not showing

### ISSUE THREAD

**Issue by iamalive-now** [gantry/gantry5#2930]
Hi
I've just adapted the message for the error page in Gantry and nothing shows up on the front end of my joomla 4 test site
Regards
David

**Comment by mahagr** [comment]
You should probably turn on debugging and check the logs to see what's going wrong.

**Comment by iamalive-now** [comment]
Sorry - I'm not doing this full time anymore, so I'm a bit slow.
The inspectors has the following:
Uncaught Error: UIkit 2.x requires jQuery
    at uikit.min.js:2
    at uikit.min.js:2
wrong:1329 Uncaught ReferenceError: jQuery is not defined
    at wrong:1329
pagePreloader.js:146 Uncaught ReferenceError: jQuery is not defined
    at pagePreloader.js:146
wrong:1343 Uncaught ReferenceError: jQuery is not defined
    at wrong:1343

I am using uikit from inspire.com - no idea what version that makes it, but apparently it's the wrong one and no jquery either? Hmm. Is that foreign particles do you think?

**Comment by mahagr** [comment]
jQuery isn't loaded by Joomla 4 anymore. You can still load it manually from Frameworks Atom.

**Comment by iamalive-now** [comment]
Ah! That explains it - thanks Matias. Regards, David

**Comment by iamalive-now** [comment]
Sorry to be dim - how do you mean load it manually from Frameworks Atom?

**Comment by iamalive-now** [comment]
I have removed every particle and every atom from the Gantry Error page outline, so a blank page showing the default system error message. The only thing left in the template layout were the "System Messages" and "Page content" particles, as well as the offcanvas section. So something in the basic error template must be different.
I copied the original again and removed everything one by one. Not until I reached the stage above, did the message show.
I then started with the empty template and rebuilt it one by one, from the bottom up.  And that's where I found the culprit: a page preloader particle. Interestingly enough, the page preloader works on all other pages!
Anyway, I have a solution and someone to inform of the problem.
Thank you.
